desktop/core/src/desktop/js/apps/admin/Configuration/Configuration.scss

```diff
@@ -0,0 +1,90 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// 'License'); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an 'AS IS' BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@import '../../../components/styles/variables';
+
+.antd.cuix {
+  .config-component {
+    background-color: $fluidx-gray-100;
+    padding: 24px;
+
+    .config__section-header {
+      color: $fluidx-gray-700;
+  }
+
+    .config__main-item {
+      .config--last-heading,
+      .config__main-item--heading {
+          font-size: $font-size-xl;
+          font-weight: 300;
+      }
+
+      .config--last-heading {
+        padding: 4px 0 8px 0;
+      }
+
+        padding: 16px 0 8px 16px;
```

Comment:
if these styles(padding, bg-color,border) belongs  to the config__main-item class add it on top itself instead of adding before closing of this class